lib/vtex/mappings/category_mapping.rb

```diff
@@ -0,0 +1,16 @@
+module Vtex
+  class CategoryMapping
+    include Kartograph::DSL
+
+    kartograph do
+      mapping Category
+
+      property :parentId, scopes: [:read]
+      property :id, scopes: [:read]
+      property :name, scopes: [:read]
+      property :hasChildren, scopes: [:read]
+      property :url, scopes: [:read]
+      property :children, scopes: [:read]
```

Comment:
question: how do you map the Children to be a Category ??? I mean, this only add a property `children` but it doesn't specify that the children is an instance or have the same properties as the Category?? does this work automatically? Please review this, children should be another Category (through the Category Mapping)